tests/test_frontend_vgg.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import annotations
+
+import torch
+from torch import nn
+
+from i6_models.parts.frontend import (
+    VGG4LayerActFrontendV1,
+    VGG4LayerActFrontendV1Config,
+)
+
+
+def test_conformer_vgg_layer_act_frontend_v1():
+    torch.manual_seed(42)
+
+    def get_output_shape(
+        batch,
+        time,
+        time_padding,
```

Comment:
What is this `time_padding` used for? I don't think you need that.

Reply:
idea was to have a sequence with padding and one without... so that I can check if the padding somehow messes something up

Reply:
I don't think you need that as an argument. You would simply create a minibatch where there are different seqs of different lengths. Then you have some with padding, maybe some without.